### ISSUE TITLE
Image watermark does not handle scale factors greater than 1.0

### ISSUE THREAD

**Issue by mdmcconnell** [pdfcpu/pdfcpu#1023]
I am running pdfcpu on the command line under wsl Ubuntu. This behaviour is occuring under the recent commit https://github.com/pdfcpu/pdfcpu/commit/b9b3d0fca12d9570a60ef7dd3527971bc13b8f71.

We can imagine a situation where it would be nice to have a watermark fill the entire background of a page, but it is not precisely the same dimensions.  Scaling it over 1.0 would allow some bleed at edges to fill the entire sheet, but it appears that for scalefactor > 1.0, pdfcpu defaults to 1.0 abs.  On the attached file, try 

`pdfcpu watermark add -mode image -- "pink.png" "rotation:0, scalefactor:1.2" dummy.pdf out.pdf`
[dummy.pdf](https://github.com/user-attachments/files/17992727/dummy.pdf)

[pink](https://github.com/user-attachments/assets/a79295e8-84c5-467b-97da-b8314b3c12b5)

**Comment by GitHubRulesOK** [comment]
@mdmcconnell 
I dont mean to interfere but since the image is one unit in both directions if you want it to bleed off edgest to fith height you only have to use that as the scale, try:

pdfcpu watermark add -mode image -- "pink.png" "scale:841 abs, rot:0" dummy.pdf out.pdf

**Comment by mdmcconnell** [comment]
Good point - I didn't think about trying that approach; looks like it works.  Though IMO scale > 1.0 ought to be a valid option as well - or cause an error instead of silently ignored.

**Comment by GitHubRulesOK** [comment]
Just an observation and thus NOT official

That HERE "relative 1.0" will use just the page size width as the pixel s width for placement.
Whilst the pixel image can be any shape such as diagonal or many times as high as wide
It is by default considered a single square object so "Relative" will stretch to fit 595 wide and 595 high centred by default.

"Absolute" is considered as "relative" to the pixel thus placed at scale which for one unit scaled up by 595 will be same as 1.0 relative to page!
